README.md

```diff
@@ -35,4 +35,5 @@ Name|Description|Default value
 ---|---|---
 repository.names | Comma separated list of repository names. Must be the same lenght as repository.urls | Maven Central
 repository.urls | Comma separated list of repository URLs. Must be the same length as repository.names | http://repo1.maven.org/maven2
+licenseServiceUrl | An optional URL of a license service. <br> If not provided, only file- and pom-based license resolution will be performed | *null*
```

Comment:
Maybe put something like "If not provided, only exception and alias sanitisers will be used". I'm not excatly sure what do you mean by file-based license resolution.